test/integration_test/migration_dr_actions_test.go

```diff
@@ -644,6 +646,577 @@ func testDRActionFailbackMonthlyScheduleTest(t *testing.T) {
+	log.FailOnError(t, err, "error retrieving statefulsets from %s namespace", elasticsearchNamespace)
+	Dash.VerifyFatal(t, len(destStatefulsets.Items), 1, fmt.Sprintf("Expected 1 statefulset in destination in %s namespace", elasticsearchNamespace))
+	Dash.VerifyFatal(t, *destStatefulsets.Items[0].Spec.Replicas, sourceStatefulsetReplicas, fmt.Sprintf("Expected %d replica in destination in %s namespace", sourceStatefulsetReplicas, elasticsearchNamespace))
+
+	// Verify that both mysql and elasticsearch applications are not running on the source cluster
+	err = setSourceKubeConfig()
+	log.FailOnError(t, err, "failed to set kubeconfig to source cluster: %v", err)
+
+	sourceDeployments, err = apps.Instance().ListDeployments(mysqlNamespace, metav1.ListOptions{})
+	log.FailOnError(t, err, "error retrieving deployments from %s namespace", mysqlNamespace)
+	Dash.VerifyFatal(t, len(sourceDeployments.Items), 1, fmt.Sprintf("Expected 1 deployment in source in %s namespace", mysqlNamespace))
+	Dash.VerifyFatal(t, *sourceDeployments.Items[0].Spec.Replicas, 0, fmt.Sprintf("Expected 0 replica in source in deployment in %s namespace", mysqlNamespace))
+
+	sourceStatefulsets, err = apps.Instance().ListStatefulSets(elasticsearchNamespace, metav1.ListOptions{})
+	log.FailOnError(t, err, "error retrieving statefulsets from %s namespace", elasticsearchNamespace)
+	Dash.VerifyFatal(t, len(sourceStatefulsets.Items), 1, fmt.Sprintf("Expected 1 statefulset in source in %s namespace", elasticsearchNamespace))
+	Dash.VerifyFatal(t, *sourceStatefulsets.Items[0].Spec.Replicas, 0, fmt.Sprintf("Expected 0 replica in source in sts in %s namespace", elasticsearchNamespace))
+
+	//////////////////////////////////////////////////////
+	// Create reverse migrationschedule using storkCtl //
+	////////////////////////////////////////////////////
+
+	err = setDestinationKubeConfig()
+	log.FailOnError(t, err, "failed to set kubeconfig to destination cluster: %v", err)
+
+	// Initialize reversemigrationschedule and reverse schedule policy.
+	reverseSchedulePolicyArgs := make(map[string]map[string]string)
+	reverseSchedulePolicyArgs["reverse-migrate-every-5m"] = map[string]string{"policy-type": "Interval", "interval-minutes": "5"}
+	reverseMigrationScheduleArgs := make(map[string]map[string]string)
+	reverseMigrationScheduleArgs[instanceID] = map[string]string{
+		"purge-deleted-resources": "",
+		"schedule-policy-name":    "reverse-migrate-every-5m",
+	}
+
+	// Create schedulePolicies using storkCtl.
+	cmd = storkctl.NewCommand(factory, os.Stdin, &outputBuffer, os.Stderr)
+	for schedulePolicyName, customArgs := range reverseSchedulePolicyArgs {
+		cmdArgs := []string{"create", "schedulepolicy", schedulePolicyName, "--kubeconfig", destinationKubeConfigPath}
+		executeStorkCtlCommand(t, cmd, cmdArgs, customArgs)
+	}
+	log.Info("created reverse schedule policy")
+
+	// Create migration schedule.
+	reverseMigrationScheduleName := "reverse-failback-modified-secondary-replicas"
```

Comment:
change the name of this schedule as we are not modifying replicas

Reply:
Done.